compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -552,6 +554,9 @@ class MapAllocatorCache {
 
     Entries[I].invalidate();
 
+    if (I == LastUnreleasedEntry)
+      LastUnreleasedEntry = Entries[LastUnreleasedEntry].Prev;
+
```

Comment:
What if we insert an entry which has been released? Now it only happens when MTE is enabled if I remember correctly

A solution for this is that we update `LastUnreleasedEntry` in store()